src/dtx/dtx_internal.h

```diff
@@ -57,12 +57,16 @@ enum dtx_operation {
 #undef X
 
 /* DTX RPC input fields */
+/* clang-format off */
 #define DAOS_ISEQ_DTX							\
 	((uuid_t)		(di_po_uuid)		CRT_VAR)	\
 	((uuid_t)		(di_co_uuid)		CRT_VAR)	\
 	((uint64_t)		(di_epoch)		CRT_VAR)	\
+	((uint32_t)		(di_version)		CRT_VAR)	\
+	((uint32_t)		(di_padding)		CRT_VAR)	\
```

Comment:
can you put di_version after di_flags, then you do not need padding?

btw: is this server-server RPC? does it need rpc version change?

Reply:
> can you put di_version after di_flags, then you do not need padding?
di_flags is array.


> btw: is this server-server RPC? does it need rpc version change?
It is server side RPC, no need version change since we do not support interoperability among different versions of servers.